client/src/components/v2/OverlayMessage.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react';
+import { Button, Card, FormControl, InputGroup } from 'react-bootstrap';
+import { useMessageState, sendMessage } from 'api/v2/camera';
+
+/**
+ * Text field for sending messages to the rider via the active overlay.
+ *
+ * @returns {React.Component} Component
+ */
+export default function OverlayMessage() {
+  const {
+    message,
+    received,
+    sending,
+    setMessage,
+    setSending,
+  } = useMessageState();
+
+  const handleSendMessage = useCallback(() => {
+    if (sendMessage(message)) setSending(true);
```

Comment:
Another reason why I'd like `sendMessage` as part of the same hook is so we can put `setSending(true)` inside `sendMessage`.